packages/terra-folder-tree/src/FolderTree.module.scss

```diff
@@ -0,0 +1,7 @@
+@import '~terra-mixins/lib/Mixins';
```

Comment:
Do we need Terra mixins?

Reply:
Not sure, this is just a placeholder file to get started on FolderTree development and the content of this file was copied from the equivalent setup for file path: https://github.com/cerner/terra-framework/pull/1771/files#diff-213a5eefceecea61e9ec5bbdb472d503d5193b21a7f7e655af877752445bd241R1